### PR TITLE
Feature/normalize data

### PR DESCRIPTION
**Pull request by roesslerj**

### CODE REVIEW

File: src/main/java/de/retest/web/WebElementPeer.java [hunk 1]
```diff
@@ -100,14 +100,21 @@ public OutlineAttribute retrieveOutline() {
 	protected MutableAttributes retrieveStateAttributes() {
 		final MutableAttributes state = new MutableAttributes();
 		for ( final String attribute : AttributesProvider.getInstance().getAttributes() ) {
-			final String attributeValue = webData.get( attribute );
+			final String attributeValue = normalize( webData.get( attribute ) );
```
Comment (diba1013) [inline src/main/java/de/retest/web/WebElementPeer.java]: Shouldn't we do this to all attributes (i.e inside the constructor)?

File: src/test/java/de/retest/web/WebElementPeerTest.java [hunk 2]
```diff
@@ -44,4 +44,9 @@ void filled_web_data_should_result_in_outline_with_corresponding_values() throws
 		assertThat( rectangle.getHeight() ).isEqualTo( 4.0 );
 	}
 
+	@Test
+	void normalize_should_remove_apostrophe() {
```
Comment (diba1013) [inline src/test/java/de/retest/web/WebElementPeerTest.java]: Theoretical what if `""`, `"\""`, `"    \""`, `"\"   "`.

File: src/main/java/de/retest/web/WebElementPeer.java (hunk 1 above)
Comment (roesslerj) [reply src/main/java/de/retest/web/WebElementPeer.java]: Not so sure about this. I don't know how future Attributes will be implemented and for which technologies. Would refrain from generalizing too much...

File: src/test/java/de/retest/web/WebElementPeerTest.java (hunk 2 above)
Comment (roesslerj) [reply src/test/java/de/retest/web/WebElementPeerTest.java]: Blanks should be trimmed. But the other cases ... would wait for them to occur in practice, to have a better understanding.

**Pull request merged**